app/views/static/_home_cover.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="cover">
+  <div class="cover-image" style="background-image: url(<%= asset_path 'static/home-cover-04.jpg' %>)"></div>
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12 col-sm-7">
+        <div class="cover-text">
+          <h1 class="cover-text__top">Discover the greatest works of art ever painted.</h1>
+          <h2 class="cover-text__bottom">Supercharge your study session with curated lesson plans designed to help you learn things faster and remember them even longer.</h2>
+        </div>
+      </div>
+      <div class="visible-xs col-xs-12 signup-form">
```

Comment:
This form should probably be copied from the devise registration file which already works. https://github.com/eruditionapp/eruditionapp/blob/master/app/views/devise/registrations/new.html.erb